src/Layouts/footer/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
```

Comment:
the extensions of the files everywhere you use jsx, should be .jsx - Footer.jsx, Header.jsx etc